arithmetization/src/main/java/net/consensys/linea/zktracer/opcode/gas/projector/Sha3.java

```diff
@@ -51,7 +55,7 @@ public long largestOffset() {
 
   @Override
   public long linearPerWord() {
-    return linearCost(Trace.GAS_CONST_G_KECCAK_256_WORD, this.length, 32);
+    return linearCost(GAS_CONST_G_KECCAK_256_WORD, this.length, 32);
```

Comment:
You've replace 32's with `WORD_SIZE` elsewhere, those could be swapped out, too.